Doc/library/typing.rst

```diff
@@ -2254,6 +2254,32 @@ Functions and decorators
 
    See :pep:`484` for details and comparison with other typing semantics.
 
+   .. versionchanged:: 3.11
+      Overloaded functions can now be introspected at runtime using :func:`get_overloads`.
+
+
+.. function:: get_overloads(func)
+
+   Return a sequence of :func:`@overload <overload>`-decorated definitions for *func*. *func* is
+   the function object for the implementation of the overloaded function.
+   For example, given the definition of ``process`` in the documentation for
+   :func:`@overload <overload>`, ``get_overloads(process)`` will return a sequence of three
```

Comment:
Something about this feels off (to reference another doc example here). But I can't find a better way to illustrate your point.